avm/src/daemon/ctrl.rs

```diff
@@ -124,6 +140,66 @@ async fn control_port(req: Request<Body>) -> Result<Response<Body>, Infallible>
   }
 }
 
+async fn handle_start(req: Request<Body>) -> Result<Response<Body>, Infallible> {
+  // Receive POST and save daemon properties
+  match get_daemon_props(req).await {
+    Ok(_) => Ok(Response::builder().status(200).body("ok".into()).unwrap()),
+    Err(err) => {
+      error!(ErrorType::DaemonStartFailed, "{:?}", err).await;
+      Ok(Response::builder().status(500).body("fail".into()).unwrap())
+    }
+  }
+}
+
+async fn get_daemon_props(req: Request<Body>) -> DaemonResult<()> {
+  let bytes = body::to_bytes(req.into_body()).await?;
+  let body: DaemonProperties = serde_json::from_slice(&bytes).unwrap();
+  might_dump_files(&body)?;
+  DAEMON_PROPS
+    .set(DaemonProperties {
+      clusterId: body.clusterId,
+      agzB64: body.agzB64,
+      deployToken: body.deployToken,
+      domain: body.domain,
+      dockerfileB64: body.dockerfileB64,
```

Comment:
why did we change the capitalization? It is usually capitalized, but also this might cause a bug in other `deploy` endpoints and the CLI that still expect the `DockerfileB64` instead of `dockerfileB64`

Reply:
Hmm I did it since it's true that it is capitalize, it is still a variable name? To avoid unnecessary change in this PR renaming it in several places will revert it.